### PR TITLE
pipeline recommendataion updates

### PR DESCRIPTION
**Pull request by melindaloubser1**
- closes https://github.com/RasaHQ/rasa-demo/issues/489
- Update pipeline recommendations for rasa > 1.8

### CODE REVIEW

**Comment by melindaloubser1** [comment]
@erohmensing @ArjaanBuijk I just saw this. https://github.com/RasaHQ/rasa-demo/pull/448 - seems to be stale but should it be revived?

**Comment by erohmensing** [comment]
@ArjaanBuijk can you take over the review of this one? #448 would be ideal if the content still makes sense, i think it went stale bc 1.8 was released.

**Comment by ArjaanBuijk** [comment]
I will close this. The whole pipeline logic has changed, and it is just as easy to start over at this point.

**Comment by akelad** [comment]
wait i'm confused, was this not melinda trying to adjust it to the new stuff?

**Comment by melindaloubser1** [comment]
Yes, this was me adjusting it to new stuff but with minimal changes unlike #448 - Arjaan are you planning to reopen that one then?

**Comment by akelad** [comment]
no i think arjaan accidentally closed this... @ArjaanBuijk? I think melinda you can re-open

File: data/nlu/nlu.md [hunk 1]
```diff
@@ -544,6 +544,7 @@
 - whats the diff between [rasa](product) and [rasa x](product)
 - when should i use [rasa](product) and when should i use [rasa x](product)
 - what is the difference between [rasa open source](product) and [rasa x](product)
+- what is the difference between [rasa open source](rasa) and [rasa x](product)
```
Comment (ArjaanBuijk) [inline data/nlu/nlu.md:547]: Is this a correct entity annotation?

Comment (melindaloubser1) [reply data/nlu/nlu.md:547]: thanks for catching that!

Comment (melindaloubser1) [reply data/nlu/nlu.md:547]:
```suggestion
- what is the difference between [rasa open source](product) and [rasa x](product)
```

**Pull request merged**